src/DependencyInjection/Configuration.php

```diff
@@ -636,6 +679,40 @@ private function getCloudflareHttpDispatcherNode()
         return $node;
     }
 
+    private function getFastlyHttpDispatcherNode()
+    {
+        $treeBuilder = new TreeBuilder('http');
+
+        // Keep compatibility with symfony/config < 4.2
+        if (!method_exists($treeBuilder, 'getRootNode')) {
+            $node = $treeBuilder->root('http');
```

Comment:
according to composer.json, we only support symfony 4.4 or newer, so we don't need this anymore (i guess we did not clean up the other places yet. we should - but lets not do it in the same pull request as its not related)